src/python/grpcio/grpc/_cython/_cygrpc/grpc.pxi

```diff
@@ -172,10 +172,14 @@ cdef extern from "grpc/grpc.h":
     GRPC_ARG_INTEGER
     GRPC_ARG_POINTER
 
-  ctypedef struct grpc_arg_value_pointer:
-    void *address "p"
+  ctypedef struct grpc_arg_pointer_vtable:
     void *(*copy)(void *)
     void (*destroy)(void *)
+    int (*cmp)(void *, void *)
+
+  ctypedef struct grpc_arg_value_pointer:
+    void *p
```

Comment:
I was under the impression that we're trying to keep with more-Python-ish style names. If not then feel free to ignore.